pkg/scheduler/objects/preemption.go

```diff
@@ -915,6 +935,11 @@ func preemptPredicateCheck(plugin api.ResourceManagerCallback, ch chan<- *predic
 		}); err == nil {
 			result.success = true
 			result.index = -1
+		} else {
+			log.Log(log.SchedPreemption).Debug("Normal predicate check failed",
```

Comment:
Shouldn't we log this an error?

Reply:
No, not required as it may flood the logs when we have many predicates to run. At times, there could be. Moreover, predicates check is not an error as it indicates that there is no resources even after the preemption.